### ISSUE TITLE
Mp malus for nekochief missing

### ISSUE THREAD

**Issue by loooph** [samlee405/dofus-lab#94]
Nekochief on dofuslab:
```
    300 Vitality
    40 Wisdom
    50 Power
    1 AP
    400 Initiative
    10 Fire Resistance
    10 % Fire Resistance
    6 AP Parry
    10 Lock
    10 Critical Resistance
```
[Nekochief on dofus.com](https://www.dofus.com/en/mmorpg/encyclopedia/equipment/14000-nekochief):
```
251 to 300 Vitality
31 to 40 Wisdom
31 to 50 Power
1 AP
-1 MP
301 to 400 Initiative
7 to 10 Fire Resistance
7 to 10% Fire Resistance
4 to 6 AP Parry
7 to 10 Lock
7 to 10 Critical Resistance
```
It is not just visual. The mp malus does not get applied.

**Comment by loooph** [comment]
Same issue for  Augilol's Tippet with the ap malus.

**Comment by lightyearsaway** [comment]
Also known: Hogmeiser's Worn Boots, Jackanapes, Royal Tofu Crown; will be fixed in #81

**Comment by lightyearsaway** [comment]
Fixed in #81
